docs/source/migrations/0.7.0/neo4j-migrations.rst

```diff
@@ -0,0 +1,77 @@
+###############
+Neo4j migration
+###############
+
+
+This note describes the necessary steps to migrate a single instance from `Neo4j <https://neo4j.com/>`__ 3.5.x to `Neo4j <https://neo4j.com/>`__ 4.0.x
+
+Upgrading to neo4j 4.0.x from a 3.5.x version
+=============================================
+
+-  Backup:
+
+The neo4j database is persisted in a docker volume (declared as stellio-entity-neo4j-storage in the entity-service ``docker-compose.yml``).
+
+We need to make a backup of this volume that can be used if the migration goes wrong.
+
+To list the docker volumes:
+
+.. code-block:: shell
+
+    docker volume ls
+
+We will use the entity-service_stellio-entity-neo4j-storage volume (please consider working with your own volume name if you changed it in the entity-service ``docker-compose.yml``).
+
+To locate the volume mountpoint:
+
+.. code-block:: shell
+
+    docker volume inspect entity-service_stellio-entity-neo4j-storage
+
+The mountpoint is the path of the ``data`` directory generated by the neo4j container.
+
+Make a copy of this directory.
+
+.. code-block:: shell
+
+    cp -r $neo4j_volume/_data $destination_folder
```

Comment:
Eventually, someone will ask you: what is $neo4j_volume? what is $destination_folder?

(aka you can't pop up variables without explaining first what they are)